BrentOzarULTD/SQL-Server-First-Responder-Kit
# Issue 1407/parlevjo2

Enhancement #1407.

Changes proposed in this pull request:
 - Check if the owner of a database is an existing user
Sometimes if you execute sp_helpdb you see databases which have no current database owner.
Mostly because the owner of the database was an administrator which has left the company, eg:

name | db_size | owner
--------- | --------- | ---------
dbname | 2284.13 MB | \~~ UNKNOWN ~~

This check reports those databases so you can give the database a proper owner like the 'sa' user.

How to test this code:
 - Create a new Windows account. 
 - Create a new SQL account for this Windows account. 
 - Create a test database with this SQL account as owner of the test database.
 - Delete the new SQL account.
 - Delete the new Windows account.
 - Execute sp_helpdb to see that the owner is Unknown.
 - Execute sp_blitz on this sql instance.

Has been tested on (remove any that don't apply):
 - SQL Server 2008 R2
 - SQL Server 2012
